src/main/resources/static/js/status-handler.js

```diff
@@ -0,0 +1,159 @@
+/*
+This js file intend to keep the status information about the current selected back-end
+and depending weather the back end status has changed and or is online or not will
+enable or disable different kinds of status icons/fields.
+It may also fore page refresh pages, remove subscriptions from subscription list
+and reload subscriptions in subscription list depending on the back-end status.
+*/
+var pagesForStatusIndication = "subscriptions test-rules ei-info login";
+
+var statusType = {
+    success: "alert-success",
+    info: "alert-info",
+    warning: "alert-warning",
+    danger: "alert-danger"
+};
```

Comment:
These are constants right? I would love to see them uppercase. Also, maybe we would even have a file with all sorts of constants?

Reply:
In Java uppercase would be prefered, in javascript this is debated.

Reply:
okay 👍